networkx/algorithms/find_euler_path.py

```diff
@@ -0,0 +1,223 @@
+import networkx as nx
+
+""" 
+Find an eulerian path
+"""
+#    Copyright (C) 2015 by
+#    Edwardo S. Rivera <edwardo.rivera@upr.edu>
+#    Humberto Ortiz <humberto@hpcf.upr.edu>
+#    BSD license. 
+#    All rights reserved.
+
+
+__author__ = """\n""".join(['Edwardo Rivera <edwardo.rivera@urp.edu>','Humberto Ortiz <humberto@hpcf.upr.edu>'])
+
+__all__ = ['find_eulerian_path'] 
+
+def find_eulerian_path(G):
```

Comment:
So, how I should write the authors?  I don't want to touch code from someone else :)